contrib/lang/ess/README.md

```diff
@@ -37,20 +37,26 @@ Start an `R` inferior REPL process with <kbd>SPC m i</kbd>.
 
 Send code to inferior process commands:
 
-    Key Binding     |                 Description
---------------------|------------------------------------------------------------
-<kbd>SPC m b</kbd>  | send buffer and keep code buffer focused
-<kbd>SPC m B</kbd>  | send buffer and switch to REPL in insert mode
-<kbd>SPC m d</kbd>  | send region or line and step (debug)
-<kbd>SPC m D</kbd>  | send function or paragraph and step (debug)
-<kbd>SPC m l</kbd>  | send line and keep code buffer focused
-<kbd>SPC m L</kbd>  | send line and switch to REPL in insert mode
-<kbd>SPC m r</kbd>  | send region and keep code buffer focused
-<kbd>SPC m R</kbd>  | send region and switch to REPL in insert mode
-<kbd>SPC m t</kbd>  | send thing at point (function) and keep code buffer focused
-<kbd>SPC m T</kbd>  | send thing at point (function) and switch to REPL in insert mode
-<kbd>CTRL+j</kbd>   | next item in REPL history
-<kbd>CTRL+k</kbd>   | previous item in REPL history
+    Key Binding      |                 Description
+---------------------|------------------------------------------------------------
+<kbd>SPC m b</kbd>   | send buffer and keep code buffer focused
+<kbd>SPC m B</kbd>   | send buffer and switch to REPL in insert mode
+<kbd>SPC m d</kbd>   | send region or line and step (debug)
+<kbd>SPC m D</kbd>   | send function or paragraph and step (debug)
+<kbd>SPC m l</kbd>   | send line and keep code buffer focused
+<kbd>SPC m L</kbd>   | send line and switch to REPL in insert mode
+<kbd>SPC m r</kbd>   | send region and keep code buffer focused
+<kbd>SPC m R</kbd>   | send region and switch to REPL in insert mode
+<kbd>SPC m t</kbd>   | send thing at point (function) and keep code buffer focused
+<kbd>SPC m T</kbd>   | send thing at point (function) and switch to REPL in insert mode
+<kbd>SPC m c n</kbd> | next knitr/sweave chunk
+<kbd>SPC m c p</kbd> | previous knitr/sweave chunk
+<kbd>SPC m c e</kbd> | send knitr/sweave chunk and keep buffer focused
+<kbd>SPC m c g</kbd> | send knitr/sweave chunk and switch to REPL in insert mode
```

Comment:
prefer `mcE` to follow the convention

also maybe `mcc` (no focus of REPL) and `mcC` (focus REPL) would be more easy.